### PR TITLE
feat(workspaceName): add workspace_name to bot object

### PR DESCRIPTION
**Pull request by hazelguo**
This PR includes:
- workspace_name change
- `ButtonPropertyItemObjectResponse` from simon
- `comment_id` change from maybe clara

### CODE REVIEW

File: src/api-endpoints.ts [hunk 1]
```diff
@@ -4681,6 +4683,13 @@ type LastEditedTimeDatabasePropertyConfigResponse = {
   name: string
 }
 
+type ButtonDatabasePropertyConfigResponse = {
```
Comment (cwang314) [inline src/api-endpoints.ts:4686]: @hazelguo this looks handwritten - if you don't add this, what happens to the types in this file? we shouldn't be exposing this property in the SDK yet (and probably not ever tbh)

File: src/api-endpoints.ts [hunk 2]
```diff
@@ -7379,6 +7406,7 @@ type BlockObjectRequest =
               link_to_page:
                 | { page_id: IdRequest; type?: "page_id" }
                 | { database_id: IdRequest; type?: "database_id" }
+                | { comment_id: IdRequest; type?: "comment_id" }
```
Comment (cwang314) [inline src/api-endpoints.ts:7409]: this is fine - keep these in

**Review by cwang314 — CHANGES_REQUESTED** [review]

File: src/api-endpoints.ts (hunk 1 above)
Comment (hazelguo) [reply src/api-endpoints.ts:4686]: I didn't handwrite anything. Everything is from `notion build-public-api-js-sdk --path=../notion-sdk-js/src/`

**Review by cwang314 — APPROVED** [review]
thanks for making requested changes!

**Pull request merged**